beacon-chain/rpc/eth/validator/structs.go

```diff
@@ -17,3 +19,7 @@ type SubmitAggregateAndProofsRequest struct {
 type SubmitSyncCommitteeSubscriptionsRequest struct {
 	Data []*shared.SyncCommitteeSubscription `json:"data" validate:"required,dive"`
 }
+
+type ProduceSyncCommitteeContributionResponse struct {
+	Data *shared.SyncCommitteeContribution
```

Comment:
`json` tag is missing. The name might be generated correctly as `data` by default, but let's be explicit here.

Reply:
this this is still missing